config.json

```diff
@@ -1,5 +1,8 @@
 {
   "PORT": 8080,
   "NODE_ENV": "development",
-  "SALT_WORK_FACTOR": 10
+  "SALT_WORK_FACTOR": 10,
+  "accessTokenTtl":"15m",
```

Comment:
Can you change all of this new fields with for exemple : ACCES_TOKEN_TTL, REFRESH_TOKEN_TTL etc..